client/my-sites/plugins/marketplace/constants.js

```diff
@@ -0,0 +1,2 @@
+export const MARKETPLACE_FLOW_ID = 'marketplace_flow';
+export const ANALYTICS_UI_LOCATION_MARKETPLACE_DOMAIN_SELECTION = 'marketplace_domain_selection';
```

Comment:
The following file `client/my-sites/plugins/marketplace/index.tsx` seems to be empty - should we remove it? Sorry for commenting here, but it seems like GH won't allow me to comment on an empty file 😉

Reply:
Hahah yup thanks! I was in the middle of removing this and somehow lost track.